torchvision/prototype/transforms/_geometry.py

```diff
@@ -256,9 +256,7 @@ def _extract_params_for_v1_transform(self) -> Dict[str, Any]:
         params = super()._extract_params_for_v1_transform()
 
         if not (params["fill"] is None or isinstance(params["fill"], (int, float))):
-            raise ValueError(
-                f"{type(self.__name__)}() can only be scripted for a scalar `fill`, but got {self.fill} for images."
-            )
+            raise ValueError(f"{type(self.__name__)}() can only be scripted for a scalar `fill`, but got {self.fill}.")
```

Comment:
I know it's not from this PR but

> type(self.__name__)

isn't this always str? Should this just be `self.class.__name__` or just `type(self)`?

Reply:
Oh yeah that is a typo. Should be

```suggestion
            raise ValueError(f"{type(self).__name__}() can only be scripted for a scalar `fill`, but got {self.fill}.")
```